rest-client/src/main/java/io/apicurio/registry/client/RegistryRestClientImpl.java

```diff
@@ -81,7 +81,7 @@
  */
 public class RegistryRestClientImpl implements RegistryRestClient {
 
-    private final RequestHandler requestHandler;
+    private final RequestExecutor requestHandler;
```

Comment:
Rename the field as well?  Since it's now an executor and not a handler...

Reply:
Whoops, good catch.